anagrams_strings.rb

```diff
@@ -0,0 +1,29 @@
+puts "Digite uma frase: "
+phrase1 = gets.chomp.delete(' ').split('')
+
+puts "Digite outra frase: "
+phrase2 = gets.chomp.delete(' ').split('')
+
+count1 = 0
+count2 = 0
+
+phrase1.each_with_index do |char, position|
+	if phrase2.include?(char)
+		count1+=1
+	end
+end
```

Comment:
[Sugestão]

```suggestion
phrase1.each_with_index do |char, position|
  count1+=1	if phrase2.include?(char)
end
```

Você pode usar o if inline quando a expressão é pequena.

https://www.ruby-forum.com/t/beginner-if-statement-in-one-line/142781 

E lembre-se sempre que o uso do each_with_index só faria sentido se precisasse saber o index, nesse caso não precisamos, podemos usar diretamente o char, então, não precisamos do `position` , pode usar somente o método `each`.

Verifique sobre `each` vs `each_with_index`.